sdk/core/azure-core-amqp/src/impl/rust_amqp/amqp/message_receiver.cpp

```diff
@@ -1,17 +1,23 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
-// Enable declaration of strerror_s.
-#define __STDC_WANT_LIB_EXT1__ 1
+// cspell: words amqpmessagereceiver amqpmessagereceiveroptions amqp Amqp
```

Comment:
Nit: `words` -> `ignore`, or maybe `words` works but `ignore` is what is documented. Also, it should be case-insensitive so you don't need both "amqp" and "Amqp". Or maybe only `ignore` is case-insensitive.

Reply:
For the C++ repo, we match case sensitive because we discovered that case insensitive matches missed a large number of spelling errors.

Reply:
And FWIW, [`cspell: words`](https://cspell.org/configuration/document-settings/#words) adds the specified words to the local dictionary, `cspell: ignore` says to ignore the listed words.

But you're right, ignore is probably more correct.